Jib-gradle not working on OpenShift

# Expected Behavior

```
It should run as successful
```

# Actual Behavior

```
Its failing.
```

# Steps to Reproduce the Problem

1. Use the steps mentioned in readme

Logs
```
{"level":"info","ts":1582729480.7104826,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"KO_DATA_PATH\" does not exist or is empty"}
FAILURE: Build failed with an exception.
* What went wrong:
Failed to load native library 'libnative-platform.so' for Linux amd64.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
{"level":"info","ts":1582729480.8973973,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"KO_DATA_PATH\" does not exist or is empty"}
2020/02/26 15:04:49 Skipping step because a previous step failed
time="2020-02-26T15:04:41.41110257Z" level=warning msg="No HTTP secret provided - generated random secret. This may cause problems with uploads if multiple registries are behind a load-balancer. To provide a shared secret, fill in http.secret in the configuration file or set the REGISTRY_HTTP_SECRET environment variable." go.version=go1.11.2 instance.id=fb6b0671-f43e-4b15-973f-9ed5ed6d8529 service=registry version=v2.7.1 
time="2020-02-26T15:04:41.411179603Z" level=info msg="redis not configured" go.version=go1.11.2 instance.id=fb6b0671-f43e-4b15-973f-9ed5ed6d8529 service=registry version=v2.7.1 
time="2020-02-26T15:04:41.411334605Z" level=info msg="Starting upload purge in 26m0s" go.version=go1.11.2 instance.id=fb6b0671-f43e-4b15-973f-9ed5ed6d8529 service=registry version=v2.7.1 
time="2020-02-26T15:04:41.420766785Z" level=info msg="using inmemory blob descriptor cache" go.version=go1.11.2 instance.id=fb6b0671-f43e-4b15-973f-9ed5ed6d8529 service=registry version=v2.7.1 
time="2020-02-26T15:04:41.421131829Z" level=info msg="listening on [::]:5000" go.version=go1.11.2 instance.id=fb6b0671-f43e-4b15-973f-9ed5ed6d8529 service=registry version=v2.7.1 
```

# Additional Info

--- a/jib-gradle/jib-gradle.yaml
+++ b/jib-gradle/jib-gradle.yaml
@@ -42,6 +42,7 @@ spec:
       # Runs the Gradle Jib build.
       gradle jib \
         --stacktrace --console=plain \
+        -g gradle-user-home \
         --init-script=/tekton/home/init-script.gradle \
         -Duser.home=/tekton/home \
         -Dgradle.user.home=/tekton/home/.gradle \

